vexim/adminuseraddsubmit.php

```diff
@@ -158,7 +158,7 @@
       mail("{$_POST['localpart']}@{$_SESSION['domain']}",
         vexim_encode_header(sprintf(_("Welcome %s!"), $_POST['realname'])),
         "$welcome_message",
-        "From: {$_SESSION['localpart']}@{$_SESSION['domain']}\r\nMIME-Version: 1.0\r\nContent-Type: text/plain; charset=utf-8\r\nContent-Transfer-Encoding: 8bit\r\n");
```

Comment:
> Sorry, I thought it was the destination. We should put the localpart in `$_SESSION[]`

Or a full email address, since I doubt we're using localpart for anything else than composing the email address. It's up to you.